Test/test0/Focus.bpl

```diff
@@ -0,0 +1,33 @@
+// RUN: %boogie /vcsDumpSplits "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+// RUN: %diff Ex.split.-1.bpl Ex.split.-1.bpl
```

Comment:
What does this command do?  It seems to be diffing a file against itself.

Reply:
I just wanted to check that this file is generated and I did not want to assume other OS level commands.